os/binfmt/libxipelf/xipelf.c

```diff
@@ -108,32 +108,32 @@ static int xipelf_loadbinary(FAR struct binary_s *binp)
 	binp->sizes[BIN_TEXT] = binp->flash_region_end - binp->flash_region_start;
 
 	/* zero out the bss... */
-	for (uint8_t *bss = uspace.bss_start; bss < uspace.bss_end; bss++) {
+	for (uint8_t *bss = uspace.bss_start; bss < (uint8_t *)uspace.bss_end; bss++) {
```

Comment:
`uint8_t *bss = uspace.bss_start;` Here is no typecasting.
But `bss < (uint8_t *)uspace.bss_end;` Here needs? It looks strange.
bss_start and bss_end have different type??